codebase/src/java/disco/org/openlvc/disco/pdu/emissions/EmitterBeam.java

```diff
@@ -128,23 +207,17 @@ public void from( DisInputStream dis ) throws IOException
 		this.targets.clear();
 		for( int i = 0; i < numberOfTargets; i++ )
 		{
-			TrackJamData record = new TrackJamData();
-			record.from( dis );
-			targets.put( record.getTarget(), record );
+			TrackJamData targetRecord = new TrackJamData();
```

Comment:
The linter was complaining because the local `record` field was shadowing a member variable that was named the same.